api/scripts/update-users-with-shouldValidateTermsOfService-to-true.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash');
+const Bookshelf = require('../lib/infrastructure/bookshelf');
+
+const BookshelfUser = require('../lib/infrastructure/data/user');
+
+async function updateShouldValidateTermsOfServiceWhenUsersHaveNotSamlIdAndNotAssociatedWithSchoolingRegistration() {
+  const subquery = Bookshelf.knex
+    .select('users.id')
+    .from('users')
+    .leftJoin('schooling-registrations', 'users.id', 'schooling-registrations.userId')
+    .where({
+      'users.samlId': null,
+      'schooling-registrations.userId': null
+    });
+
+  const foundUsers = await BookshelfUser
+    .query((qb) => {
+      qb.update({ shouldValidateTermsOfService: true }, ['id'])
+        .whereIn('id', subquery);
+    })
+    .fetchAll();
+
+  return _.map(foundUsers.models, 'id');
+}
+
+async function main() {
+  console.log('Start update "shouldValidateTermsOfService" users table column to true.');
+
+  try {
+    const updatedUserIds = await updateShouldValidateTermsOfServiceWhenUsersHaveNotSamlIdAndNotAssociatedWithSchoolingRegistration();
+    console.log(`Succesfully update ${updatedUserIds.length} records.`);
```

Comment:
Il manque des lettres :

```suggestion
    console.log(`Successfully updated ${updatedUserIds.length} records.`);
```

Voir aussi plus haut la suggestion de ne pas renvoyer la liste des `id` mais directement le nombre de lignes affectées.

Reply:
Corrigé